odata-protocol/1 Introduction.md

```diff
@@ -294,11 +294,9 @@ one or both of them may differ from convention.
 ## ##subsec Transient Entities
 
 Transient entities are instances of an entity type that are 
-the result of transforming entities of that entity type
-and only exist within a single payload.
+dynamically generated on request and only exist within a response payload.
 They cannot be updated and consequently possess neither a stable entity-id nor an update URL.
-A transient entity may have a read URL, which may be used to apply the same transformation to
-the current state of the underlying entity or entities.
+A transient entity may have a read URL, which generates a new transient entity using the same algorithm.
```

Comment:
```suggestion
A transient entity may have a read URL, which regenerates the transient entity using the same algorithm.
```
I prefer "regenerates" over "generates a new", because the read URL remains the same.

Reply:
The URL remains the same, and potentially returns a different transient entity.